packages/instant/src/components/ui/text.tsx

```diff
@@ -47,6 +49,8 @@ export const Text = styled(PlainText)`
     ${props => (props.display ? `display: ${props.display}` : '')};
     ${props => (props.letterSpacing ? `letter-spacing: ${props.letterSpacing}` : '')};
     ${props => (props.textTransform ? `text-transform: ${props.textTransform}` : '')};
+    ${props => (props.marginRight ? `margin-right: ${props.marginRight}` : '')};
+    ${props => (props.marginLeft ? `margin-right: ${props.marginLeft}` : '')};
```

Comment:
You seem to have added these but not used them? Also, we try to keep positional and layout props in the `Container` component (yes, it does make it a bit of a catch-all), and keep text-related props to `Text`. So the way I would do this here, if you did need it, would be to wrap the `Text` in a `Container` with margin.

Reply:
Yes, I was originally using just `Text` for the error icon, and not `Container`, so I needed this change.  I ended up using a `Container` but forgot to remove this.  Glad this was brought up though -- now I better understand the pattern of making `Text` props only related to `text` specific properties. 👍